dlt/helpers/dbt/dbt_utils.py

```diff
@@ -135,15 +128,10 @@ def run_dbt_command(
         runner_args = (global_args or []) + [command] + args  # type: ignore
 
         with dbt.logger.log_manager.applicationbound():
-            try:
```

Comment:
Looks like with this change we're not supporting `dbt<1.5` anymore. We could write a note in the docs that `1.5.2` (looking at `test_runner_dbt_versions.py`) is the minimum tested/supported version.

Reply:
right! the fact is we do not support <1.5 for a long time already...